accessibility/accessibility.go

```diff
@@ -16,7 +16,7 @@ import (
 // Given invalid input (non-integers, integers outside of the range), the user
 // will continue to be reprompted until a valid input is given, ensuring that
 // the return value is always valid.
-func PromptInt(prompt string, low, high int) int {
+func PromptInt(w io.Writer, r io.Reader, prompt string, low, high int) int {
```

Comment:
@caarlos0 You're changing the signature of some functions. Isn't this a problem regarding backwards compatibility?

Reply:
but yeah, will update to keep compat